README.md

```diff
@@ -34,7 +34,7 @@
 </p>
 
 <p align="center">
-  <strong>Train a quantum computer the same way as a neural network.</strong>
+  The definitive open-source Python framework for quantum programming. Built by researchers, for research.
   <img src="https://raw.githubusercontent.com/PennyLaneAI/pennylane/master/doc/_static/header.png#gh-light-mode-only" width="700px">

```

Comment:
This image has a few implications since there requires a dark and light mode version (which the current image supports both). Tarik suggested this image:
![PL logo](https://github.com/user-attachments/assets/4a902c36-75b2-443c-b917-a827a60d8d28)
Though, it will require a version for light-mode.

To avoid delaying this PR further, I will merge as-is and aim to update this image with @tarik-elkhateeb shortly after.